### DISCUSSION TITLE
Processsing time

### DISCUSSION THREAD

**Discussion by rosenph** [QSMxT/QSMxT#59] [General]
I ran the QSMxT program successfully on a 6-echo sequence from a Philips 3T, with 124 slices and an in-plane resolution of 0.6mm. The processing was done by a Dell Optiplex 7090  i7 4-core computer running Ubuntu 22.04. The problem is that it took 2.5 hours to complete!
Is this normal?
Is there anything I can do to shorten the reconstruction time?

**Comment by stebo85** [comment]
Dear @rosenph,

Great to hear that the processing worked for you :) The processing times you see are normal for QSM and there is currently not much you can do to speed it up without compromising quality.

If you want to half the processing time at the cost of more artifacts you could deactivate the two-pass optimization and run only a single pass. You could do this with the flag --single_pass in run_2_qsm.py

We are currently working on two ways to speed up the processing hopefully without compromising the quality - so keep an eye out for future updates :)

Cheers
Steffen

**Comment by rosenph** [comment]
Dear Steffen,

Thank you for your reply. I'll just have to be patient.

Best regards,

Philip Rosen.

‫בתאריך יום ו׳, 29 ביולי 2022 ב-0:43 מאת ‪Steffen Bollmann‬‏ <‪
***@***.***‬‏>:‬
